docs/releasing-workflow-instructions.md

```diff
@@ -55,6 +55,10 @@ then after the review we do a couple of additions and re-release staging ``DAJIA
 
 ### Instructions to generate a staging release
 
+0. THIS IS VERY IMPORTANT:
```

Comment:
I think this is updated doc icon  📝   :-))

And BTW, perhaps we should merge this with ``[skip CI]`` to avoid all the CI ...